pkgs/servers/urserver/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchurl
+, autoPatchelfHook
+, bluez
+, libX11
+}:
+
+stdenv.mkDerivation rec {
+  pname = "urserver";
+  version = "3.6.0.745";
+
+  src = fetchurl {
+    url = "https://www.unifiedremote.com/static/builds/server/linux-x64/745/urserver-${version}.tar.gz";
+    sha256 = "1ib9317bg9n4knwnlbrn1wfkyrjalj8js3a6h7zlcl8h8xc0szc8";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    bluez
+    libX11
+  ];
+
+  installPhase = ''
+    install -m755 -D urserver $out/bin/urserver
+    cp -r remotes $out/bin/remotes
+    cp -r manager $out/bin/manager
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://www.unifiedremote.com/";
+    description = "The one-and-only remote for your computer";
+    licence = licences.unfree;
```

Comment:
```suggestion
    license = licenses.unfree;
```

Reply:
Oh wow, thanks for catching that. I was staring at that for ages and I didn't see why it failed. Not sure why it worked on my end though.

Reply:
That's because building locally doesn't evaluate `meta`. That's something ofborg does as a part of its evaluations!